helm/templates/user-configmap.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-user-configmap
+data:
+    mongo-users.js: |
+      print("Started Adding the Users.");
+      db = db.getSiblingDB("admin");
+
+      // works for version < 5.x.x (based on mongo version 4.4.18)
+      const mongo_root_password = _getEnv('MONGO_INITDB_ROOT_PASSWORD');
```

Any reason for naming it with "INITDB"? 
Can we simply call these environment variables without that part?
```suggestion
      const mongo_root_password = _getEnv('MONGO_ROOT_USER_PASSWORD');
```

---

If this name has nothing to do with the environment variables that we create on services, it is probably fine.

---

so INITDB had to do with the root username. But now since we are disabling the authentication flag as of now, we can make that change. Will change it in the latest push.